libs/fixtures/data/electionFamousNames2021/cdf-election.ts

```diff
@@ -0,0 +1,244 @@
+            Content: 'Democrat',
+          },
+        ],
+      },
+      Abbreviation: {
+        '@type': 'InternationalizedText',
+        Text: [
+          {
+            '@type': 'BallotDefinition.LanguageString',
+            Language: 'en',
+            Content: 'D',
+          },
+        ],
+      },
+    },
+  ],
+
+  // Definition of "geo-political units" (e.g. states, counties, districts, precincts, etc)
+  GpUnit: [
+    {
+      '@type': 'BallotDefinition.ReportingUnit',
+      '@id': 'state-hamilton',
+      Type: 'state',
+      Name: {
+        '@type': 'InternationalizedText',
+        Text: [
+          {
+            '@type': 'BallotDefinition.LanguageString',
+            Language: 'en',
+            Content: 'State of Hamilton',
+          },
+        ],
+      },
+      ComposingGpUnit: ['county-franklin'],
+    },
+    {
+      '@type': 'BallotDefinition.ReportingUnit',
+      '@id': 'county-franklin',
+      Type: 'county',
+      Name: {
+        '@type': 'InternationalizedText',
+        Text: [
+          {
+            '@type': 'BallotDefinition.LanguageString',
+            Language: 'en',
+            Content: 'Franklin County',
+          },
+        ],
+      },
+      // TODO what's the relationship between counties and districts?
+      ComposingGpUnit: ['district-1'],
+    },
+    {
+      '@type': 'BallotDefinition.ReportingUnit',
+      '@id': 'district-1',
+      Type: 'district',
+      Name: {
+        '@type': 'InternationalizedText',
+        Text: [
+          {
+            '@type': 'BallotDefinition.LanguageString',
+            Language: 'en',
+            Content: 'District 1',
+          },
+        ],
+      },
+      ComposingGpUnit: ['precinct-23', 'precinct-22'],
+    },
+    {
+      '@type': 'BallotDefinition.ReportingUnit',
+      '@id': 'precinct-23',
+      Type: 'precinct',
+      Name: {
+        '@type': 'InternationalizedText',
+        Text: [
+          {
+            '@type': 'BallotDefinition.LanguageString',
+            Language: 'en',
+            Content: 'North Lincoln',
+          },
+        ],
+      },
+    },
+    {
+      '@type': 'BallotDefinition.ReportingUnit',
+      '@id': 'precinct-22',
+      Type: 'precinct',
+      Name: {
+        '@type': 'InternationalizedText',
+        Text: [
+          {
+            '@type': 'BallotDefinition.LanguageString',
+            Language: 'en',
+            Content: 'South Lincoln',
+          },
+        ],
+      },
+    },
+  ],
+
+  // Required metadata
+  GeneratedDate: '2023-01-23T00:00:00Z',
+  Issuer: 'VotingWorks',
+  IssuerAbbreviation: 'VX',
+  VenderApplicationId: 'VxDefine', // App that generated this file
+  Version: '1.0.0', // CDF version
+  SequenceStart: 1, // Report number (out of a series of reports - don't think this applies for this use case but is required)
+  SequenceEnd: 1, // Total number of reports in series
+
+  // Extra fields not in CDF
+  // TODO should these be specified in this file?
```

Comment:
Do we want to extend the CDF to include these fields?

Reply:
Since we use these fields to layout the ballot and interpret scans, I think we have to include them somehow.

Reply:
IIRC we can put the seal somewhere in the physical layout?

For now, we need to include them as this is the only way we can configure, but longterm we may want to move adjudication/threshold configuration into the app.